packages/astro/src/content/runtime.ts

```diff
@@ -56,7 +55,9 @@ export function createGetCollection({
 		} else if (collection in dataCollectionToEntryMap) {
 			type = 'data';
 		} else {
-			return warnOfEmptyCollection(collection);
+			// eslint-disable-next-line no-console
+			console.warn(`The collection **${collection}** does not exist or is empty. Ensure a collection directory with this name exists.`);
```

Comment:
Should this use the `logger` so it respects the configured `logLevel`?

Reply:
This is the runtime so there is no access to the logger. Same reason we use console.warn in src/runtime/*